documentation/3.3/content/release-notes.md

```diff
@@ -47,6 +48,12 @@ draft: false
 
 ### Change log
 
+#### Operator 3.3.5
+
+* Resolved an issue related to collecting logs for failed Model in Image domains that used auxiliary images ([#2627](https://github.com/oracle/weblogic-kubernetes-operator/pull/2627)).
```

Comment:
Should we also mention PR 2629 in addition to PR 2627?

Reply:
Okay :)